adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/routes/AssetsResource.kt

```diff
@@ -50,7 +49,6 @@ import javax.ws.rs.core.Response
 @Produces(MediaType.APPLICATION_JSON)
 @Timed
 @ApplicationScoped
-@GZIP
```

Comment:
did not found GZIP support in RESTeasy Reactive yet, asked in the [Quarkus Zulip Chat](https://quarkusio.zulipchat.com/#narrow/stream/187030-users/topic/Resteasy.20Reactive.3A.20GZIP).